SECURITY.md

```diff
@@ -0,0 +1,5 @@
+## Security-Related Bugs
+
+DO NOT POST SECURITY-RELATED BUGS IN PUBLIC MAILING LISTS OR DISCUSSION FORUMS.
+
+Security-related bugs should be reported only via email to security@plone.org. Security issues in Zope, Plone or its add-ons can be reported this way, and the Security Team will contact the relevant maintainer if necessary.
```

Comment:
```suggestion
Security-related bugs should be reported only via email to (security@plone.org)[mailto:security@plone.org]. Security issues in Zope, Plone or its add-ons can be reported this way, and the Security Team will contact the relevant maintainer if necessary.
```

Reply:
I very rarely write markdown. Is my suggested change needed at all?

Reply:
https://www.markdownguide.org/basic-syntax/ suggests a "shortcut" of enclosing an email address with `<>` to create a mailto link. I'll do that.

Reply:
The builtin MarkDown renderer of my editor renders the email address as an hyperlink, even when it is plain text and with no special formatting applied.

Reply:
> The builtin MarkDown renderer of my editor renders the email address as an hyperlink, even when it is plain text and with no special formatting applied.

Yes, I saw that myself when looking at the rendered version of the previous commit. Oh well.